Test Host/PickerController.m

```diff
@@ -44,6 +48,9 @@ - (void)viewDidLoad
 
     datePicker = [[UIDatePicker alloc] initWithFrame:CGRectMake(30, 215, 260, 35)];
     datePicker.datePickerMode = UIDatePickerModeDate;
+    if(@available(iOS 13.4, *)) {
+        datePicker.preferredDatePickerStyle = UIDatePickerStyleWheels;
```

Comment:
Might help to explicitly rename `datePicker` to `wheelDatePicker` or something to make the distinction more clear.